cuav/image/scanner.c

```diff
@@ -65,16 +63,29 @@ struct scan_params {
     bool blue_emphasis;
 };
 
-static const struct scan_params scan_params_640_480 = {
-	min_region_area : 8,
-        max_region_area : 400,
-        min_region_size_xy : 2,
-        max_region_size_xy : 30,
-        histogram_count_threshold : 50,
-        region_merge : 1,
-        save_intermediate : false,
-        blue_emphasis : false
-};
+#ifdef _MSC_VER
```

Comment:
probably better to have just one declaration, with comments on fields

  8, // min_region_area

Reply:
Yep, fixed